api/qanpb/filters.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package qan;
+
+option go_package = "qanpb";
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+// Filters serves list of names of metrics.
+service Filters {
+  // GetFilters gets map of metrics names.
+  rpc GetFilters(FiltersRequest) returns (FiltersReply) {
```

Comment:
Method `XXX` should use `XXXRequest` and `XXXResponse`.
I propose to call method `Get` (`Filters` is already present in service name) and use URL `/v1/qan/Filters/Get`. That matches other APIs.